src/style.css

```diff
@@ -0,0 +1,122 @@
+:root {
+  --color-main: #060b35;
+  --color-secondary: #616070;
+  --color-blue: #00acdc;
+  --color-light: #fff;
+  --color-border: #f3f3f3;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid var(--color-border);
+  border-radius: 5px;
+}
+
+.card__img {
+  display: flex;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main);
+}
+
+.card__description {
+  align-self: start;
+  margin-bottom: 16px;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: var(--color-secondary);
+}
+
+.card__review,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.card__review {
+  margin-bottom: 24px;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+  align-self: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__review-count {
+  font-weight: 400;
+  font-size: 10px;
+  color: var(--color-main);
+}
+
+.card__price {
+  margin-bottom: 16px;
+  align-items: center;
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: 12px;
+  color: var(--color-secondary);
+}
+
+.card__price-sum {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: var(--color-main);
+}
+
+.card__button {
+  padding: 12px 70px 12px 69px;
```

Comment:
More correctly to remove horizontal paddings and make this button fullwidth in some other way